// [python quickstart](https://grpc.io/docs/quickstart/python.html#run-a-grpc-application)
// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. server.proto

// backend.proto
syntax = "proto3";

service Backend {
    rpc SayHello(HelloRequest) returns (HelloResponse) {}
    rpc SayHelloAgain(HelloRequest) returns (HelloResponse) {}
}


//为了避免麻烦，特别是列表图片等　直接采用通用的字节流全部用字典打包序列化成字节流传送
message HelloRequest {
    bytes para = 1;
}

message HelloResponse {
    bytes result = 1;
}